src/app/pages/debrief/debrief.page.html

```diff
@@ -1,100 +1,94 @@
 
-  <dangerous-faults-debrief-card
-    [dangerousFaults]="pageState.dangerousFaults$ | async">
-  </dangerous-faults-debrief-card>
+    <dangerous-faults-debrief-card
+            [dangerousFaults]="pageState.dangerousFaults$ | async">
+    </dangerous-faults-debrief-card>
 
-  <serious-faults-debrief-card
-    [seriousFaults]="pageState.seriousFaults$ | async">
-  </serious-faults-debrief-card>
+    <serious-faults-debrief-card
+            [seriousFaults]="pageState.seriousFaults$ | async">
+    </serious-faults-debrief-card>
 
-  <driving-faults-debrief-card
-    [drivingFaults]="pageState.drivingFaults$ | async"
-    [drivingFaultCount]="pageState.drivingFaultCount$ | async"
-    [testCategory]="testCategory">
-  </driving-faults-debrief-card>
+    <driving-faults-debrief-card
+            [drivingFaults]="pageState.drivingFaults$ | async"
+            [drivingFaultCount]="pageState.drivingFaultCount$ | async"
+            [testCategory]="testCategory">
+    </driving-faults-debrief-card>
 
-  <eco-debrief-card
-    *ngIf="pageState.showEco$ | async"
-    [adviceGivenControl]="adviceGivenControl"
-    [adviceGivenPlanning]="adviceGivenPlanning">
-  </eco-debrief-card>
+    <eco-debrief-card
+            *ngIf="pageState.showEco$ | async"
+            [adviceGivenControl]="adviceGivenControl"
+            [adviceGivenPlanning]="adviceGivenPlanning">
+    </eco-debrief-card>
 
-  <vehicle-checks-card-cat-b
-    *ngIf="isCategoryBTest()">
-  </vehicle-checks-card-cat-b>
+    <vehicle-checks-card-cat-b
+            *ngIf="isCategoryBTest()">
+    </vehicle-checks-card-cat-b>
 
-  <vehicle-checks-card
-    *ngIf="showVehicleChecksArrayCard()"
-    [category]="pageState.category$ | async"
-    [tellMeShowMeQuestions]="pageState.tellMeShowMeQuestions$ | async">
-  </vehicle-checks-card>
+    <vehicle-checks-card
+            *ngIf="showVehicleChecksArrayCard()"
+            [category]="pageState.category$ | async"
+            [tellMeShowMeQuestions]="pageState.tellMeShowMeQuestions$ | async">
+    </vehicle-checks-card>
 
-  <cpc-debrief-card
-    *ngIf="!isTerminated() && showCPCDebriefCard()"
-    [overallScore]="pageState.overallScore$ | async"
-    [question1]="pageState.question1$ | async"
-    [question2]="pageState.question2$ | async"
-    [question3]="pageState.question3$ | async"
-    [question4]="pageState.question4$ | async"
-    [question5]="pageState.question5$ | async"
-    [testOutcome]="outcome">
-  </cpc-debrief-card>
+    <cpc-debrief-card
+            *ngIf="!isTerminated() && showCPCDebriefCard()"
+            [overallScore]="pageState.overallScore$ | async"
+            [question1]="pageState.question1$ | async"
+            [question2]="pageState.question2$ | async"
+            [question3]="pageState.question3$ | async"
+            [question4]="pageState.question4$ | async"
+            [question5]="pageState.question5$ | async"
+            [testOutcome]="outcome">
+    </cpc-debrief-card>
 
+    <safety-questions-card
```

Comment:
Is this missing the check that used to happen? i.e. `*ngIf="isCatD() && pageState.showSafetyQuestions$ | async"`